README.md

```diff
@@ -111,8 +111,7 @@ These are currently used to compute properties of a minimum energy conformation
 | `SMIRNOFF Coverage Set 1` | [2019-06-25-smirnoff99Frost-coverage](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2019-06-25-smirnoff99Frost-coverage)  | Hessian calculations. |  Cl, Br, S, C, F, P, I, O, H, N |  [![Error](https://img.shields.io/badge/Status-Error-red)](https://img.shields.io/badge/Status-Error-red) |
 | `OpenFF Theory Benchmarking Single Point Energies v1.0` | [2021-09-06-theory-bm-single-points](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2021-09-06-theory-bm-single-points)  | Single Point Energy dataset for the final optimized geometries from MP2/heavy-aug-cc-pVTZ torsiondrives. |  Cl, F, C, S, O, H, N, P | [![Running](https://img.shields.io/badge/Status-Running-orange)](https://img.shields.io/badge/Status-Running-orange) |
 | `SPICE Solvated Amino Acids Single Points Dataset v1.0` | [2021-11-08-QMDataset-Solvated-Amino-Acids-single-points](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2021-11-08-QMDataset-Solvated-Amino-Acids-single-points)  | Single point energy calculation of solvated amino acids. | N, S, O, C, H | [![Running](https://img.shields.io/badge/Status-Running-orange)](https://img.shields.io/badge/Status-Running-orange) |
-| `TorsionNet500 Single Points Dataset v1.0` | [2021-11-09-TorsionNet500-single-points](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2021-11-09-TorsionNet500-single-points)  | Single point energies of final geometries of TorsionNet500 dataset. | H, O, F, S, N, Cl, C | [![Running](https://img.shields.io/badge/Status-Running-orange)](https://img.shields.io/badge/Status-Running-orange) |
-| `SPICE DES Monomers Single Points Dataset v1.0` | [2021-11-15-QMDataset-DES-monomers-single-points](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2021-11-15-QMDataset-DES-monomers-single-points | Single point energy calculation of DES monomers. | I, C, Br, P, Cl, H, S, O, F, N | [![Running](https://img.shields.io/badge/Status-Running-orange)](https://img.shields.io/badge/Status-Running-orange) |
```

Comment:
What are these removals for?

Reply:
Ohh! I misread them as duplicate, will correct it.